File: spec/code_samples/Node unirest/keys/get.js
var unirest = require("unirest");

var req = unirest("GET", "http://ec2-3-222-208-106.compute-1.amazonaws.com:8086/keys");

req.query({
  "exchange": "SOME_STRING_VALUE",
  "activated": "SOME_BOOLEAN_VALUE"
});


req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

File: spec/code_samples/Python python3/exchange-keys/get.py
import http.client

conn = http.client.HTTPConnection("ec2-3-222-208-106.compute-1.amazonaws.com:8086")

conn.request("GET", "/exchange-keys?exchange=SOME_STRING_VALUE&activated=SOME_BOOLEAN_VALUE")

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))